### ISSUE TITLE
Failure with io.mpv.Mpv.yml

### ISSUE THREAD

**Issue by fastrizwaan** [flathub/flatpak-builder-lint#29]
io.mpv.Mpv builds fine and runs fine on my debian system. flathub buildbot gives not so helpful message.  
```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions io.mpv.Mpv.yml
 in dir /srv/buildbot/worker/build-x86_64-5/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions io.mpv.Mpv.yml'
 using PTY: False
{
    "errors": [
        "module-pipewire-jack-runtime-source-git-branch",
        "module-luajit-source-git-branch"
    ],
    "warnings": [
        "module-uchardet-cmake-no-debuginfo",
        "finish-args-contains-both-x11-and-wayland",
        "module-x265-buildsystem-is-plain-cmake",
        "module-x265-cmake-no-debuginfo",
        "module-x265-source-md5-deprecated"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
elapsedTime=0.337740
```

**Comment by hfiguiere** [comment]
"module-pipewire-jack-runtime-source-git-branch",
       "module-luajit-source-git-branch"

In both case a `branch` is used for git source instead of a pinned tag / commit. This is not allowed.

As for pipewire-jack, you should just remove it altogether, it's in the 22.08 runtime.

For luajit, pin a commit.

**Comment by fastrizwaan** [comment]
Ok, thanks.
